tests/localclient/test_get_surface_obs.py

```diff
@@ -159,7 +159,9 @@ def test_gcwerks_retrieval():
         "data_type": "timeseries",
     }
 
-    assert metadata == expected_metadata
+    for key, expected_value in expected_metadata.items():
```

Comment:
I've updated the expected metadata dict as this loop just checks the dictionary against itself.